src/ocm/components/clusterConfiguration/review/ReviewStep.tsx

```diff
@@ -40,6 +61,17 @@ const ReviewStep = ({ cluster }: { cluster: Cluster }) => {
     }
   };
 
+  if (!hasSupportLevel) {
+    return (
+      <ErrorState
+        title={'Feature support levels not available'}
+        content={`OpenShift version ${
+          cluster.openshiftVersion || ''
+        } does not currently include feature support levels.`}
```

Comment:
You are right, this message should not be seen by users.
In any case, we tried to make it as clear as possible in case some developer has this issue due to some problem in their environment configuration. The message was checked with Allison.